Formula/commitlint.rb

```diff
@@ -0,0 +1,22 @@
+require "language/node"
+
+class Commitlint < Formula
+  desc "📓 Lint commit messages"
+  homepage "https://commitlint.js.org/#/"
+  url "https://registry.npmjs.org/commitlint/-/commitlint-17.1.2.tgz"
+  sha256 "aca659da30c4c3e25483fca656f60b35b521f397f1edf5fb4d5eaaf2a65d83e3"
+  license "MIT"
+  head "https://github.com/conventional-changelog/commitlint.git", branch: "master"
+
+  depends_on "node"
+
+  def install
+    system "npm", "install", *Language::Node.std_npm_install_args(libexec)
+    bin.install_symlink Dir["#{libexec}/bin/*"]
+  end
+
+  test do
+    assert "commitlint", "--version"
```

Comment:
```suggestion
    assert bin/"commitlint", "--version"
```

Same below. Plus it would be nice to check this output, you can use `shell_output` and `assert_match` functions.

Reply:
I add a test for an example config, in case of working _without_ linting errors, the `assert_match` should refer to empty.